core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/tags/SimpleTagQueryParser.java

```diff
@@ -217,6 +238,7 @@ public static boolean isRegExp(String tagValuesQuery) {
                         .flatMap(e -> dataAccess.findMetricsByTagNameValue(tenantId, e.getTagName(), e.getTagValues())
                                 .compose(new TagsIndexRowTransformerFilter<>(metricType))
                                 .compose(new ItemsToSetTransformer<>())
+                                .doOnNext(metricIds -> logQuery(tenantId, e, metricIds))
```

Comment:
Yes, it is acceptable. The threshold I put in place is arbitrary.  This logging is primarily intended for debugging. If we have a situation where we are having problems with HTTP requests timing out, then we can turn on debug logging in this class so that we always have the query info logged.